console-framework-client/src/main/java/io/axoniq/console/framework/AxoniqConsoleConfigurerModule.java

```diff
@@ -249,15 +260,16 @@ public Builder initialDelay(Long initialDelay) {
         }
 
         /**
-         * The thread pool's size that is used for various tasks, such as sending metrics to AxonIQ Console.
+         * The thread pool's size that is used for reporting tasks, such as sending metrics to AxonIQ Console.
          * Defaults to {@code 2}.
          *
-         * @param threadPoolSize The thread pool size
+         * @param reportingThreadPoolSize The thread pool size
          * @return The builder for fluent interfacing
          */
-        public Builder threadPoolSize(Integer threadPoolSize) {
-            BuilderUtils.assertPositive(threadPoolSize, "AxonIQ Console threadPoolSize must be positive");
-            this.threadPoolSize = threadPoolSize;
+        public Builder reportingThreadPoolSize(Integer reportingThreadPoolSize) {
```

Comment:
We didn't release with the `threadPoolSize` builder yet, right? If not, then we're not dealing with a breaking change.